src/azure-cli-core/azure/cli/core/_identity.py

```diff
@@ -22,6 +22,8 @@
     ManagedIdentityCredential
 )
 
+from azure.core.exceptions import ClientAuthenticationError
```

Comment:
Please delay this import to void performance downgrade. See https://github.com/Azure/azure-cli/pull/13843.